content/zh/guide/v11/api-reference.md

```diff
@@ -0,0 +1,202 @@
+---
+title: API 参考
+description: 了解 Preact 模块导出的所有函数
+---
+
+# API 参考
+
+此页为您提供所有导出函数的速查表。
+
+---
+
+<toc></toc>
+
+---
+
+## Component
+
+`Component` 是用于创建有状态 Preact 组件的基类。
+
+渲染器会自动管理并按需创建组件，不会直接实例化。
+
+```js
+import { Component } from 'preact';
+
+class MyComponent extends Component {
+	// (见下)
+}
+```
+
+### Component.render(props, state)
+
+所有组件必须提供 `render()` 函数，其参数为组件的当前属性 (props) 与状态 (state)，返回值则是虚拟 DOM 元素 (JSX 元素)、Array，或 `null`。
+
+```jsx
+import { Component } from 'preact';
+
+class MyComponent extends Component {
+	render(props, state) {
+		// props 等同于 this.props
+		// state 等同于 this.state
+
+		return <h1>Hello, {props.name}!</h1>;
+	}
+}
+```
+
+请参阅[组件文档](/guide/v10/components)来了解其概念及使用方法。
+
+## render()
+
+`render(virtualDom, containerNode, [replaceNode])`
```

Comment:
The third arg, `replaceNode`, is no longer supported: https://github.com/preactjs/preact-www/pull/1298/commits/24178641bfb926b25f1f2bfa71b6bca6c2adc9d2